ci/properties/rails-lint.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Rails - Build and Run Linters",
+    "description": "Build Rails application and run linters",
+    "iconName": "ruby",
+    "categories": ["Ruby", "Rails"]
```

Comment:
we are not detecting Rails in scout yet, so make sure to follow this nomenclature once we add support in scout

Reply:
@NinadKavimandan @aparna-ravindra wouldn’t it be better to add Rails detection to Scout now itself? Or are you planning that at a later point in time?

Reply:
Yes, I am working on it. Will raise a PR asap.